src/Components/Profile/Subcomponents/GalleryView.tsx

```
@@ -6,15 +6,20 @@ import { useAppState } from '#components/AppShell/hooks/useAppState'
 
 import type { Item } from '#types/Item'
 
+const extensionMap = new Map([
+  ['image/jpeg', '.jpg'],
+  ['image/png', '.png'],
+])
+
 export const GalleryView = ({ item }: { item: Item }) => {
   const [index, setIndex] = useState(-1)
   const appState = useAppState()
   const images =
-    item.gallery?.map((i, j) => ({
-      src: appState.assetsApi.url + `${i.directus_files_id.id}.jpg`,
-      width: i.directus_files_id.width,
-      height: i.directus_files_id.height,
-      index: j,
+    item.gallery?.map(({ directus_files_id: { id, type, width, height } }, index) => ({
+      src: `${appState.assetsApi.url}${id}${extensionMap.get(type) ?? ''}`,
```

I don't like the silent failure with rendering when there is an incompatible file type.

Suggestion: Add above (below `extensionMap`:
```
const getExtension = (type: string) => {
  const extension = extensionMap.get(type)

  if (extension) return extension

  throw new Error(`Unsupported file type: ${type}`)
}
```

and use it in the template string.